django_tgbot/bot_api_user.py

```diff
@@ -359,6 +384,10 @@ def setChatPermissions(self, chat_id, permissions):
 
     def exportChatInviteLink(self, chat_id):
         return self.request_and_result(create_params_from_args(locals()), bool)
+    
+    def createChatInviteLink(self , chat_id , name : str = None , expire_date : int = None , member_limit : int = 5 , 
+                            creates_join_request : bool  = False ) :
+        return self.request_and_result(create_params_from_args(locals()), CreateChatInviteLink )
```

Comment:
```suggestion
    
    def createChatInviteLink(self, chat_id, name : str = None, expire_date : int = None, member_limit : int = 5, 
                            creates_join_request : bool  = False) :
        return self.request_and_result(create_params_from_args(locals()), CreateChatInviteLink)
```

Reply:
In addition to removing the spaces as above, please also make sure the parameters are exactly like they are defined in Telegram API. In their API, there is no default value for `member_limit` or `creates_join_request`.